src/components/Cart.js

```diff
@@ -3,139 +3,185 @@ import { connect } from 'react-redux';
 import {
   deleteOrderProductsThunk,
   updateOrderProductThunk,
-  setOrderProductsThunk,
   setOrdersThunk,
-  setProductsThunk,
-  setUsersThunk
-} from '../store';
+
+} from '../redux/store';
+import axios from 'axios';
+import { updateOrderThunk, updateProductThunk } from '../redux/thunks';
+import { Link } from 'react-router-dom';
 
 class _Cart extends React.Component {
   constructor(props) {
     super();
+    this.state = {
+      id: '',
+      userId: '',
+      status:'',
+      total:'',
+      items:[]
+    };
     this.deleteItem = this.deleteItem.bind(this);
     this.updateItem = this.updateItem.bind(this);
+    this.updateOrder = this.updateOrder.bind(this);
+    this.updateInventoryFromDelete = this.updateInventoryFromDelete.bind(this);
+    this.updateInventoryFromQuantity = this.updateInventoryFromQuantity.bind(this);
+    this.changePage = this.changePage.bind(this)
+  }
+  async componentDidMount(props) {
+    await this.props.setOrders()
```

Comment:
Generally you shouldn't be `await`ing inside of a `componentDidMount`.

Reply:
Got rid of await